src/components/questions/questionsList.js

```diff
@@ -80,12 +118,12 @@ constructor(props){
         </div><div>
         {
         //IF DISPLAYED QUESTIONS IS LESS THAN TOTAL QUESTIONS THEN DISPLAY 'SHOW MORE' BUTTON
-        (this.determineDisplayed(this.props.store.questions).length <
-        this.props.store.questions.length) ?
+        (this.determineDisplayed(filteredSortedQuestions).length <
+        filteredSortedQuestions.length) ?
         <button className="button is-medium" onClick={this.seeMoreQuestionsClick}>SHOW MORE QUESTIONS</button> :
         //ELSE IF DISPLAYED QUESTIONS IS EQUAL TO TOTAL QUESTIONS SHOW 'COLLAPSE' BUTTON
-        (this.determineDisplayed(this.props.store.questions).length ===
-        this.props.store.questions.length && this.props.store.questions.length > 4) ?
+        (this.determineDisplayed(filteredSortedQuestions).length ===
+        filteredSortedQuestions.length && filteredSortedQuestions.length > 4) ?
```

Comment:
I like this method of using "length > 4" check so that you handle edge case where you are not displaying collapse button when total length is 4 and all results are shown and they don't actually need to be collapsed.